modules/backend/formwidgets/ColorPicker.php

```diff
@@ -244,6 +254,33 @@ protected function loadAssets()
      */
     public function getSaveValue($value)
     {
-        return strlen($value) ? $value : null;
+        if (!strlen($value)) {
+            return null;
+        }
+
+        switch (is_array($this->formats) ? 'all' : $this->formats) {
+            case 'cmyk':
+            case 'hex':
+            case 'hsl':
+            case 'rgb':
+                if (!preg_match($this->validationPatterns[$this->formats], $value)) {
```

Comment:
is the value already trimmed / do the regex patterns allow for untrimmed values?

Reply:
All the values should be trimmed by the frontend, the colorpicker doesn't allow extra spaces, so in theory yes, even if you type extra spaces they should be removed